lib/publish.js

```diff
@@ -36,47 +36,53 @@ module.exports = async (pluginConfig, context) => {
     await Promise.all(
       globbedAssets.map(async asset => {
         const {path, label, type, filepath} = isPlainObject(asset) ? asset : {path: asset};
-        const file = resolve(cwd, path);
-        let fileStat;
-
-        try {
-          fileStat = await stat(file);
-        } catch (_) {
-          logger.error('The asset %s cannot be read, and will be ignored.', path);
-          return;
+        const _url = asset.url
```

Comment:
I think a small remark in the `README.md` would be helpful, to point users to this feature. E.g.:

```markdown
// ...

{
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    ["@semantic-release/gitlab", {
      "gitlabUrl": "https://custom.gitlab.com",
      "assets": [
        {"path": "dist/asset.min.css", "label": "CSS distribution"},
        {"path": "dist/asset.min.js", "label": "JS distribution", "type": "generic_package"},
        {"url": "https://gitlab.com/gitlab-org/gitlab/-/blob/master/README.md"}
      ]
    }],
  ]
}

// ...

#### assets

// ...

| `url`   | This provides the ability to add links to releases. E.g. URLs to container images. | -                                    |
```